src/main/resources/evaluate_for_free_keys/idea202.evaluation.key

```diff
@@ -0,0 +1 @@
+������
```

Comment:
What are these keys? Is it wise to share them in public repo?

Reply:
@odockal Those are configuration files that are being created after you start the trial of IntelliJ Ultimate. It is the same case as other config files project works with to accept and bypass dialogs/blockers. Since anybody is capable of creating them just by clicking on a button inside IntelliJ Ultimate (no licence needed), it is safe to share them.